### PR TITLE
fixes issue#552

### PR DESCRIPTION
**Pull request by romitgithub**
https://github.com/topcoder-platform/u-bahn-app/issues/552

### CODE REVIEW

File: client/src/services/api.js [hunk 1]
```diff
@@ -34,6 +34,7 @@ export default () => {
           redirect_uri: window.location.origin,
         });
       } else if (
+        error.status === 409 &&
```
Comment (callmekatootie) [inline client/src/services/api.js:37]: @romitgithub Can you kindly let me know what this change is for...

Comment (romitgithub) [reply client/src/services/api.js:37]: To prevent any undefined error here. It was added to catch a specific case of error here.

Comment (callmekatootie) [reply client/src/services/api.js:37]: This change restricts rejecting with the error message ONLY if the status is 409 - we want to display the error for all instances (whenever a message is available). Can you kindly take another stab at this...

Comment (callmekatootie) [reply client/src/services/api.js:37]: Which error was being caught here - out of curiosity

Comment (romitgithub) [reply client/src/services/api.js:37]: @callmekatootie I think this change is not really required. I mistook it for some other bug. Should I remove this?

Comment (callmekatootie) [reply client/src/services/api.js:37]: Yep

Comment (callmekatootie) [reply client/src/services/api.js:37]: Actually - no worries... I'll remove it

Comment (romitgithub) [reply client/src/services/api.js:37]: Alright, thanks :)

**Comment by callmekatootie** [comment]
Also - don't forget to pull latest code and keep your base branch in sync

**Pull request closed without merging**